TEMPLATE/libs/map.jinja

```diff
@@ -63,4 +63,4 @@
 
 {#- Per formula post-processing of `mapdata` if it exists #}
 {%- do salt["log.debug"]("map.jinja: post-processing of 'mapdata'") %}
-{%- include tplroot ~ "/post-map.jinja" ignore missing %}
+{%- include tplroot ~ "/parameters/post-map.jinja" ignore missing %}
```

Comment:
You can use `formula_param_dir` variable defined line 9.

Reply:
oh thanks, I totaly missed it!